spec/hyacinth/digital_object/resource_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Hyacinth::DigitalObject::Resource do
+  let(:argument_names) { [:import_location, :import_method, :import_checksum, :location, :checksum, :original_filename, :file_size] }
+  let(:arguments) { argument_names.map { |s| [s.to_s, s.to_s] }.to_h }
+  let(:attribute_names) { argument_names.select { |n| !n.to_s.starts_with?('import_') } }
```

Comment:
Relying on the `import_` prefix for converting argument_names to attribute_names seems a little fragile.  This naming convention isn't enforced anywhere in the code.

Reply:
Here's a suggestion that's unrelated to the above comment.  Since the `import_*` variables technically are attributes (because of their attr_accessor definitions in the Resource class), can we change `let(:attribute_names)` to `let(:serialized_attribute_names)` and `let(:attributes)` to `let(:serialized_attributes)`?